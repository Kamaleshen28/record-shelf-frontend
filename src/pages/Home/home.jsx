import React from 'react'
import Body from '../../components/Body/body'
import Header from '../../components/Header/header'

export default function Home() {
    return(
        <div>
            <Header />
            <Body />
        </div>
    )
}