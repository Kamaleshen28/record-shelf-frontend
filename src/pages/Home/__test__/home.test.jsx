import { render} from '@testing-library/react';
import React from 'react';
import { useNavigate } from 'react-router-dom';
import Home from '../home';

jest.mock('react-router-dom');

describe('allSongs', () => {
    it('should render allSongs correctly', () => {
        const {asFragment} = render(<Home />);
        expect(asFragment()).toMatchSnapshot();
    });
});