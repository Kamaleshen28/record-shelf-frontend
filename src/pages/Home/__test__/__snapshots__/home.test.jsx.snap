// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allSongs should render allSongs correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="header"
    >
      <p>
        My 
        <span
          class="record-header"
        >
          Record
        </span>
         Shelf
      </p>
    </div>
    <div
      class="body-content"
    >
      <div
        class="body-content-container"
      >
        <p>
          :((
        </p>
        <p>
          seems a bit empty in here..
        </p>
      </div>
      <div
        class="button-container"
      >
        <button
          class="sync-button"
          data-testid="sync-button"
        >
          sync
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
